Cargo.toml

```diff
@@ -57,14 +57,13 @@ aws-lc-rs = ["aws_lc_rs"] # Alias because Cargo features commonly use `-`
 fips = ["aws_lc_rs", "aws-lc-rs?/fips", "tokio-rustls?/fips"]
 ring = ["dep:ring", "tokio-rustls?/ring"]
 server = ["dep:base64", "dep:http", "dep:httparse", "tokio/io-util"]
-simd = ["dep:simdutf8"]
```

Comment:
Could we keep (but ignore) this feature until the next semver incompatible release? This PR doesn't need to be a breaking change.

Will also require mentioning in the README.

Reply:
We already have breaking changes in `main` due to Payload now being `Sync`, I don't see a reason to hold back here.

Reply:
Should be done now.